docs/pages/guides/monorepos.mdx

```diff
@@ -167,7 +167,7 @@ registerRootComponent(App);
 
 > This new entrypoint already exists for bare projects. You only need to add this if you have a managed project.
 
-If you are using [Expo Router](/router/introduction/), define the [`EXPO_USE_METRO_WORKSPACE_ROOT`](/more/expo-cli/#environment-variables) environment variable when running the `npx expo start` command. It enables the auto server root detection for Metro.
+If you are using [Expo Router](/router/introduction/), you should not change the default entrypoint. Just define the [`EXPO_USE_METRO_WORKSPACE_ROOT`](/more/expo-cli/#environment-variables) environment variable when running the `npx expo start` command. It enables the auto server root detection for Metro.
```

Comment:
I'm not sure if this provides enough context for everyone to understand what to do. There are a couple of ongoing things that will help us remove this section fully:
- Fixing the default `expo/AppEntry` in the template, as it just doesn't work with monorepos
- Making `EXPO_USE_METRO_WORKSPACE_ROOT` turned on by default in SDK 52 (can still turn it off)

Reply:
By default entry point I mean specifically expo-router/entry. It works perfectly with the monorepo and you'll break the app if you change it for that suggested index.js. For now, maybe we could make it more explicit? 

"if you are using expo router, you should keep expo-router/entry as your entry point".

Reply:
Thanks for the suggestion, @antonyleme! Mentioning the `expo-router/entry` could also lead to broken behavior in the future if that changes. As per @byCedric's suggestion here, it's safe to mention to avoid changing the default entry point, I think. This will help keep these instructions future-proof.